explore/src/main/scala/explore/proposal/ProgramUserInvitations.scala

```diff
@@ -3,54 +3,115 @@
 
 package explore.proposal
 
+import cats.effect.IO
 import cats.syntax.all.*
-import explore.model.UserInvitation
+import crystal.*
+import crystal.react.*
+import crystal.react.hooks.*
+import explore.Icons
+import explore.components.*
+import explore.components.ui.ExploreStyles
+import explore.model.AppContext
+import explore.model.CoIInvitation
+import explore.model.IsActive
 import explore.model.reusability.given
 import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.enums.InvitationStatus
 import lucuma.react.common.ReactFnProps
+import lucuma.react.primereact.*
+import lucuma.react.primereact.Button
+import lucuma.react.syntax.*
 import lucuma.react.table.*
+import lucuma.ui.primereact.*
 import lucuma.ui.syntax.all.given
 import lucuma.ui.table.*
+import queries.common.InvitationQueriesGQL.*
 
-case class ProgramUserInvitations(users: List[UserInvitation])
+case class ProgramUserInvitations(invitations: View[List[CoIInvitation]])
     extends ReactFnProps(ProgramUserInvitations.component)
 
 object ProgramUserInvitations:
   private type Props = ProgramUserInvitations
 
-  private val ColDef = ColumnDef[UserInvitation]
+  private val ColDef = ColumnDef[CoIInvitation]
 
-  private val KeyId: ColumnId   = ColumnId("id")
-  private val EmailId: ColumnId = ColumnId("email")
+  private val KeyId: ColumnId    = ColumnId("id")
+  private val EmailId: ColumnId  = ColumnId("email")
+  private val RevokeId: ColumnId = ColumnId("revoke")
 
   private val columnNames: Map[ColumnId, String] = Map(
-    KeyId   -> "ID",
-    EmailId -> "email"
+    KeyId    -> "ID",
+    EmailId  -> "email",
+    RevokeId -> ""
   )
 
   private def column[V](
     id:       ColumnId,
-    accessor: UserInvitation => V
-  ): ColumnDef.Single[UserInvitation, V] =
+    accessor: CoIInvitation => V
+  ): ColumnDef.Single[CoIInvitation, V] =
     ColDef(id, accessor, columnNames(id))
 
-  private val columns: List[ColumnDef[UserInvitation, ?]] =
+  private def columns(active: View[IsActive], invitations: View[List[CoIInvitation]])(
```

Comment:
`deleteConfirmation` modifies the `active` parameter, but I don't see it used elsewhere. Did you want to disable the Button while active was true?

Reply:
It is used on the Invite Popup

Reply:
Maybe I'm missing something, but I only see it being used by the delete confirmation dialog, which sets it to true and then false. But I don't see anything else using it.

Reply:
Yeah deleteConfirmation needs it to call a common dialog box. That is also called in another part of the code  that does use it. I'll anyway disable the button based on `IsActive`